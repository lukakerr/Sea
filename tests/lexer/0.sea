bln var = true
num x = 5
str string = 'Hello'
str stringTwo = "Hello again"