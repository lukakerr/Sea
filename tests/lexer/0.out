[DataType B,Identifier "var",Operator Eq,Bln True,DataType N,Identifier "x",Operator Eq,Num 5,DataType S,Identifier "string",Operator Eq,Str "Hello",DataType S,Identifier "stringTwo",Operator Eq,Str "Hello again"]
